css/Components/Text.scss

```diff
@@ -12,5 +13,16 @@ $Text-font-weight: normal;
   line-height: $Text-line-height;
   font-family: $Text-font-family;
   font-weight: $Text-font-weight;
-  
+}
+
+.Text.Text--white {
+  color: $Color-white;
+}
+
+.Text.Text--uppercase {
```

Comment:
Not sure about this one, but I would say that this is also a `Link` specific thing